modules/layers/src/column-layer/column-geometry.js

```diff
@@ -0,0 +1,136 @@
+import {Geometry, uid} from '@luma.gl/core';
+
+const INDEX_OFFSETS = {
+  x: [2, 0, 1],
+  y: [0, 1, 2],
+  z: [1, 2, 0]
+};
+
+export const FILL_MODE = 'fill';
+export const STROKE_MODE = 'stroke';
```

Comment:
I am thinking to it is more accurate to call it `WIREFRAME_MODE`. Also WireframeModel, WireframeGeometry...

Reply:
`STROKE_MODE` to `WIREFRAME_MODE`?